package.json

```diff
@@ -40,6 +40,7 @@
     "test:coverage": "vitest --coverage",
     "test:types": "tsc --build tsconfig.json",
     "test": "./scripts/test.sh",
+    "prepare": "nuxi prepare playground/basic",
```

Comment:
```suggestion
```

We have `dev:prepare` in scripts

Reply:
Moved `dev:prepare` into `prepare` so it runs at the end of install script :)